src/simpleAuth.ts

```diff
@@ -43,28 +43,30 @@ export class AuthContext {
 
 export class AuthManager extends EventEmitter {
 
-    private userAuthKeyToTokensMap = new Map<string, AuthContext>(); // UserAuthKey to AuthTokens
+    userAuthKeyToTokensMap = new Map<string, AuthContext>(); // UserAuthKey to AuthTokens
 
-    constructor(private appId: string, private appPassword: string, private defaultRedirectUri: string, private scopes: string[] = []) {
+    constructor(private appId: string, private appPassword: string, private redirectUrl: string = "error", private scopes: string[] = []) {
         super();
     }
 
     // Clients of the authManager interact with it using an opaque AuthKey (string) or OID (string).  
```

Comment:
this comment block might be better interspersed with code, so its easier to conceptually map for a new reader.

Reply:
Makes sense